package/MDAnalysis/core/AtomGroup.py

```diff
@@ -3123,17 +3235,13 @@ def __init__(self, *args, **kwargs):
              individual list member. [``None``]
              Can also pass a subclass of :class:`MDAnalysis.coordinates.base.Reader`
              to define a custom reader to be used on the trajectory file.
-          *bonds*
```

Comment:
Mention in CHANGELOG that the bonds keyword went away and was replaced by guess_bonds.

For right now, catch `bonds` keyword, issue a deprecation warning ("will go away in 0.11.0"), and try to do the right thing. We then remove that code for 0.11.0.